julia/core.py

```diff
@@ -286,7 +289,11 @@ def __init__(self, init_julia=True, jl_init_path=None):
                     "Install PyCall by running the following line:\n"
                     """\tjulia -e 'Pkg.add("PyCall"); Pkg.update()'\n""")
             try:
-                self.call('pyinitialize(C_NULL)')
+                if os.name == "nt":
+                    # on windows C_NULL doesn't mean anything
+                    self.call('pyinitialize()')
```

Comment:
@stonebig, the problem here is that using `pyinitialize()` is extremely fragile in this context: if it loads a different `libpython` than the one that is currently running then everything will break.   We need to figure out what the underlying problem is.